src/Depressurizer/Database.cs

```diff
@@ -732,6 +732,7 @@ public int UpdateFromHLTB(bool includeImputedTimes)
 
             using (WebClient client = new WebClient())
             {
+                client.Encoding = System.Text.Encoding.UTF8;
```

Comment:
Would you mind changing it to the following, this way it is consistent in the code base.
```suggestion
                client.Encoding = Encoding.UTF8;
```

Reply:
done